### PR TITLE
Log server error

### PR DESCRIPTION
**Pull request by mtrinh11**

### CODE REVIEW

**Comment by mtrinh11** [comment]
fixes https://github.com/xmidt-org/argus/issues/221

File: store/transport.go [hunk 1]
```diff
@@ -217,23 +222,36 @@ func transferHeaders(w http.ResponseWriter, h http.Header) {
...
+			w.Header().Set(XmidtErrorHeaderKey, sErrorer.SanitizedError())
+		}
+
+		code := http.StatusInternalServerError
+		var statusCoder kithttp.StatusCoder
+		if errors.As(err, &statusCoder) {
+			code = statusCoder.StatusCode()
+		}
+
+		if getLogger == nil {
```
Comment (kristinapathak) [inline store/transport.go:243]: You can move this code block above the `return func...` line. 👍

File: store/transport.go [hunk 2]
```diff
@@ -217,23 +222,36 @@ func transferHeaders(w http.ResponseWriter, h http.Header) {
...
+			getLogger = func(_n context.Context) *zap.Logger {
```
Comment (kristinapathak) [inline store/transport.go:244]:
```suggestion
			getLogger = func(_ context.Context) *zap.Logger {
```

**Review by kristinapathak — COMMENTED** [review]
Looks great! Two small suggestions.

**Review by kristinapathak — APPROVED** [review]

**Pull request merged**